wpeverest/user-registration
# UR-45 Dev - PDF enhancements

### All Submissions:






### Changes proposed in this Pull Request:
Previously there was no option with `radio-image`, `image` upload and `toggle` in settings. This PR adds this settings.

### How to test the changes in this Pull Request:

1. Activate PDF form submissions addon and go to User Registration -> Settings -> PDF Submissions.
2. Verify if all of the options are working perfectly.

### Types of changes:




### Other information:




### Changelog entry

> Dev - PDF enhancements.
